From 9a9c52dc8f2a9e71ec59c3572522e278120a4e1a Mon Sep 17 00:00:00 2001
From: Tadeusz Struk <tadeusz.struk@intel.com>
Date: Wed, 9 Dec 2015 11:59:47 -0800
Subject: [PATCH 15/23] crypto: qat - select PCI_IOV when VF are enabled
Git-commit: 75910d375ecb2079b3418f8b304fd775916025e2
Patch-Mainline: v4.5
References: FATE#319820

Fix i386-randconfig-x004-12092241.
PCI_IOV needs to be selected when VFs are enabled

Reported-by: <fengguang.wu@intel.com>
Signed-off-by: Tadeusz Struk <tadeusz.struk@intel.com>
Signed-off-by: Herbert Xu <herbert@gondor.apana.org.au>
Signed-off-by: Oliver Neukum <oneukum@suse.com>
---
 drivers/crypto/qat/Kconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/crypto/qat/Kconfig b/drivers/crypto/qat/Kconfig
index d275d48..85b44e5 100644
--- a/drivers/crypto/qat/Kconfig
+++ b/drivers/crypto/qat/Kconfig
@@ -60,6 +60,7 @@ config CRYPTO_DEV_QAT_DH895xCCVF
 config CRYPTO_DEV_QAT_C3XXXVF
 	tristate "Support for Intel(R) C3XXX Virtual Function"
 	depends on X86 && PCI
+	select PCI_IOV
 	select CRYPTO_DEV_QAT
 	help
 	  Support for Intel(R) C3xxx with Intel(R) QuickAssist Technology
@@ -71,6 +72,7 @@ config CRYPTO_DEV_QAT_C3XXXVF
 config CRYPTO_DEV_QAT_C62XVF
 	tristate "Support for Intel(R) C62X Virtual Function"
 	depends on X86 && PCI
+	select PCI_IOV
 	select CRYPTO_DEV_QAT
 	help
 	  Support for Intel(R) C62x with Intel(R) QuickAssist Technology
-- 
2.1.4

